### DISCUSSION TITLE
Mathematical operations in Kconfig

### DISCUSSION THREAD

**Discussion by benjaminkelenyi** [zephyrproject-rtos/zephyr#48524] [Q&A] [answered]
Hello! 

Is there any way to make mathematical operations in the Kconfig file? 
Like: MEM_ADDRESS + FLASH_SIZE? Where MEM_ADDRESS is 0x0123456 and FLASH_SIZE is 0x654321.

Thanks,
Benjamin

**Comment by stephanosio** [comment, marked as answer]
Unfortunately, no.

**Reply by benjaminkelenyi** [reply]
workarounds?
